lib/flip_card.dart

```diff
@@ -60,6 +60,7 @@ class FlipCard extends StatefulWidget {
   final FlipCardController? controller;
   final Fill fill;
   final CardSide side;
+  final Duration? autoFlipDuration;
```

Comment:
Please documentation for this feature, ideally as a `/// comment` above the autoFlipDuration, but also in the readme

Reply:
Added to documentation.